src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/TextAnalyzer.java

```diff
@@ -151,6 +160,23 @@ public Utterance analyze(String input, Conversation conversation) throws Illegal
 		if(Logger.debug()) {
 			System.out.println("\tGreeting?Closing?: " + (stop-start) + " milliseconds");
 		}
+		
+		
+		//Sets sentence to lowercase and removes contractions, for use with spf semantic analyzer
+		//@author David S Smith
```

Comment:
Remove the @author tags scattered throughout. Unless you've made major changes to the class you don't need an @author attribution. (And if you do make major changes, the @author is usually placed in the Javadoc comment at the top of the file)